executor/point_get_test.go

```diff
@@ -162,7 +162,7 @@ func (s *testPointGetSuite) TestPointGetCharPK(c *C) {
 	tk.MustPointGet(`select * from t where a = "   ";`).Check(testkit.Rows(` `))
 }
 
-func (s *testPointGetSuite) TestIndexLookupCharPK(c *C) {
```

Comment:
@XuHuaiyu We still need to test the behavior of double read?
Maybe we need copy the tests. And one has a primary key so choose point get. And one is a normal index so choose double read.

Reply:
done